config/production.js

```diff
@@ -1,10 +1,14 @@
 module.exports = {
-  port: process.env.PORT,
-  app: {
-    url: process.env.APP_URL
+  api: {
+    port: process.env.PORT,
+    name: process.env.APP_NAME,
```

Comment:
the idea here is the ENV_NAME should always match the config path, so if you want `APP_NAME` it should either correspond to `app.name` or should be `API_NAME`.

Reply:
from recollection, this is the app name, not to do specifically with the api.

Reply:
fixed